modules/serverless-creating-subscription-admin-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-cluster-admin-eventing.adoc
+
+[id="serverless-creating-subscription-admin-web-console_{context}"]
+= Creating a subscription by using the Administrator perspective
+
+If you have cluster administrator permissions and have created a channel, you can create a subscription to connect your broker to a subscriber by using the Administrator perspective in the web console.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
+* You have cluster administrator permissions for {product-title}.
+* You have created a channel.
+* You have created a Knative service to use as a subscriber.
+
+.Procedure
+
+. In the *Administrator* perspective of the {product-title} web console, navigate to *Serverless* -> *Eventing*.
+. In the *Channel* tab, select the image:../images/kebab.png[title="Kebab"] icon for the channel that you want to add a subscription to.
```

Comment:
Since kebab is defined in the common-attributes.adoc as a replaceable, you should be able to just say:

    Options menu {kebab}

instead of:

     image:../images/kebab.png[title="Kebab"] icon